docs_user/modules/proc_migrating-tls-everywhere.adoc

```diff
@@ -0,0 +1,187 @@
+[id="migrating-tls-everywhere_{context}"]
+
+//:context: tls
+
+= TLS Everywhere
+
+_Disclaimer: the below steps were reproduced on a FreeIPA 4.10.1 server. The location of files and directories may slightly change on different versions._
+
+== Prerequisites
+
+* Check that the source deployment is using TLS Everywhere, skip this step if it's not.
+* Make sure the previous Adoption steps (if any) have been performed successfully.
+* Make sure the backend services on the new deployment are not started yet.
+
+== Variables
+
+Define the shell variables used in the following steps. The values are illustrative and refer to a single node standalone director deployment. Use values that are correct for your environment:
+
+ifeval::["{build}" != "downstream"]
+----
+IPA_SSH="ssh -i ~/install_yamls/out/edpm/ansibleee-ssh-key-id_rsa root@192.168.122.100 podman exec -ti freeipa-server-container"
+----
+
+In this example the FreeIPA instance is running on a separate host, in a container.
+endif::[]
+ifeval::["{build}" == "downstream"]
+----
+IPA_SSH="ssh -i <path_to_ssh_key> root@<freeipa-server-ip-address>"
+----
+endif::[]
+
+Define the map of compute node name, IP pairs.
+
+[subs=+quotes]
+----
+declare -A computes
+computes=(
+  ["standalone.localdomain"]="192.168.122.100"
+  # ...
+)
```

Comment:
This is an undo of what https://issues.redhat.com/browse/OSPRH-2301 achieved -- we don't want to ssh into compute nodes because there can be hundreds of them.

I don't think we should block the initial merge on this, but can we subsequently move this into edpm-ansible just like we moved everything for Nova? It looks like this is needed only for removal of certmonger and associated cleanup, so this should be quite easily doable with edpm-ansible hopefully?

Reply:
We could track this as another story under https://issues.redhat.com/browse/OSPRH-1488 and we could make verification of https://issues.redhat.com/browse/OSPRH-2301 depend on that story.

Reply:
This makes sense, it's mostly just another service to stop, so it should be easy to integrate.